### DISCUSSION TITLE
[Connect Here] Add your social profiles and follow your OPA friends

### DISCUSSION THREAD

**Discussion by peteroneilljr** [open-policy-agent/community#23] [Community]
## Use this thread to share your social media profiles and connect with the OPA Community.

Are you new to the community and looking to connect with other OPA users? This is the place for you. Use the template below to share your social media accounts and follow other members of the OPA community.

### Sharing Template

```
<!--  Add an About You section. Use the following questions or come up with your own.  -->

About Me
===
Hi, I'm OPA!
Where do you work?
What are you working on?
How long have you been using OPA?

<!--  Let people know what they can expect when following you, and link to your profiles.  -->

My Socials
===
I like to talk about x and y, follow me on these social channels.

Twitter: https://www.twitter.com/username
GitHub: https://www.github.com/username
LinkedIn: https://www.linkedin.com/in/username

```

**Comment by peteroneilljr** [comment]
About Me
===
Hi, I'm Peter! 
I work at Styra
I'm currently building the OPA community.
I've been working with OPA for 3 months! 

My Socials
===
I like to talk about Cloud Native and Community, follow me on these social channels.

Twitter: https://www.twitter.com/peteroneilljr
GitHub: https://www.github.com/peteroneilljr
LinkedIn: https://www.linkedin.com/in/peteroneilljr

**Comment by anderseknert** [comment]
About Me
===
Hi, I'm Anders.
I work at [Styra](https://www.styra.com/).
My title is "developer advocate" which means I do a little bit of anything and everything in the world of OPA :) If there's a common theme to what I do I'd say it's to help others enjoy OPA and Rego as much as I do.
I've been working on and off with OPA for the last 3 years, and full time for the last year or so.



My Socials
===
I like to talk about OPA, identity and access control, follow me on these social channels.

Twitter: https://www.twitter.com/anderseknert
GitHub: https://www.github.com/anderseknert
LinkedIn: https://www.linkedin.com/in/anderseknert

**Comment by CodaAzzurra** [comment]
About Me
===
Hello! I'm Daniel.
I work at [Styra](https://www.styra.com/).
I am Director of Customer Success, which means I run the organization that takes care of our customers.
I started using OPA when I joined Styra in early 2020 – I love it!

My Socials
===

GitHub: https://github.com/CodaAzzurra
LinkedIn: https://www.linkedin.com/in/dc600/

**Comment by ogazitt** [comment]
About Me
===
Hi, I'm Omri!
I work at [Aserto](https://aserto.com).
We're building an application / API authorization solution using OPA, as well as a set of open source tools that we think help solve some problems for the broader OPA community. A couple of examples:
- A go [package](https://github.com/aserto-dev/runtime) that make it easy to build solutions on top of the OPA runtime
- [Open Policy Registry](https://openpolicyregistry.io) (OPCR) - a CLI and registry for building, tagging, pushing, and pulling OPA policies as OCI images

I've been using OPA for a year.

My Socials
===
I like to talk about application/API authorization, and [blog](https://aserto.com/blog) pretty frequently. 

Twitter: https://www.twitter.com/omrig
GitHub: https://www.github.com/ogazitt
LinkedIn: https://www.linkedin.com/in/ogazitt

**Comment by mbrg** [comment]
About Me
===
Hi, I'm Michael!
I work at [Zenity](https://zenity.io). We help enterprises adopt low-code development while staying secure and in control.
We're building a policy and detection engine using conftest, and some additional tooling where needed (e.g. [sphinx-rego](https://github.com/zenitysec/sphinx-rego) a documentation generator for Open Policy Agent policies).
I've been using OPA for a year.

My Socials
===
I like to talk about cybersecurity / low-code and their convergence on social media and on our [blog](https://zenity.io/blog/).

Blog: 
Twitter: https://www.twitter.com/mbrg0
GitHub: https://www.github.com/mbrg
LinkedIn: https://www.linkedin.com/in/michaelbargury

**Reply by anderseknert** [reply]
Awesome! Would be nice to see Zenity added to the list of [adopters](https://github.com/open-policy-agent/opa/blob/main/ADOPTERS.md)! :)
